content/blog/2018/istio-authorization/index.md

```diff
@@ -0,0 +1,211 @@
+provides access control for services in an Istio mesh. Istio authorization is Istio’s native authorization support.
+It can be used on any platform that Istio supports. It features:
+
+* Authorization at different levels of granularity, including namespace level, service level, and method level.
+* Service-to-service and end-user-to-service authorization.
+* High performance, as it is enforced natively on Envoy.
+* Role-based semantics, which makes it easy to use.
+* High flexibility as it allows users to define conditions using
+[combination of Istio attributes](/docs/reference/config/authorization/constraints-and-properties/).
+
+In this blog post, we first elaborate Istio authorization characteristics, next we show how you could use Istio
+authorization policy in different use cases.
+
+## Istio Authorization Characteristics
+
+### RPC Level Authorization
+
+Istio authorization provides RPC level authorization. Specifically, it controls “who can access my `bookstore` service”,
+or “who can access method `getBook` in my `bookstore` service”. It is not designed to control access to application-specific
+resource instances, like access to “storage bucket X” or access to “3rd book on 2nd shelf”. Today this kind of application
+specific access control logic needs to be handled by the application itself.
+
+### Role Based Access Control with Conditions
+
+Istio authorization is Role-Based Access Control (RBAC) system. Compared to Attribute-Based Access Control (ABAC),
+RBAC has the following advantages:
+
+* **Roles allow grouping of attributes.** Roles are groups of permissions, which specifies the actions you are allowed
+to perform on a system. Users are grouped based on the roles within an organization. You can define the roles and reuse
+them for different cases.
+
+* **It is easier to understand and reason about who has access.** The RBAC concepts maps naturally to the business concepts.
+For example, a DB admin may have all access to DB backend services, while a web client may only be able to view the
+frontend service.
+
+* **It reduces unintentional errors.** RBAC policies make the originally complicated security changes easy. You won't have
+duplicate configurations in multiple places and later forget to update some of them when you need to make changes.
+
+On the other hand, Istio authorization is not a traditional RBAC system. It also allows users to define **conditions** using
+[combination of Istio attributes](/docs/reference/config/authorization/constraints-and-properties/). This gives Istio
+authorization plenty of flexibility to express complex access control policies. In fact, **the “RBAC + conditions” model
+that Istio authorization adopts, has all the benefits an RBAC system has, and supports the level of flexibility that
+normally an ABAC system provides.** We will show some examples in the [Examples section](#examples).
```

Comment:
You'll see some [Examples](#examples) below.

Reply:
Done.